tensorflow/core/BUILD

```diff
@@ -1349,60 +1349,63 @@ cc_library(
     name = "all_kernels_statically_linked",
     visibility = ["//visibility:private"],
     deps = [
-        "//tensorflow/core/kernels:array",
-        "//tensorflow/core/kernels:audio",
-        "//tensorflow/core/kernels:batch_kernels",
-        "//tensorflow/core/kernels:bincount_op",
-        "//tensorflow/core/kernels:boosted_trees_ops",
-        "//tensorflow/core/kernels:candidate_sampler_ops",
-        "//tensorflow/core/kernels:checkpoint_ops",
-        "//tensorflow/core/kernels:collective_ops",
-        "//tensorflow/core/kernels:control_flow_ops",
-        "//tensorflow/core/kernels:ctc_ops",
-        "//tensorflow/core/kernels:cudnn_rnn_kernels",
-        "//tensorflow/core/kernels:data_flow",
-        "//tensorflow/core/kernels:dataset_ops",
-        "//tensorflow/core/kernels:decode_proto_op",
-        "//tensorflow/core/kernels:encode_proto_op",
-        "//tensorflow/core/kernels:fake_quant_ops",
-        "//tensorflow/core/kernels:function_ops",
-        "//tensorflow/core/kernels:functional_ops",
-        "//tensorflow/core/kernels:grappler",
-        "//tensorflow/core/kernels:histogram_op",
-        "//tensorflow/core/kernels:image",
-        "//tensorflow/core/kernels:io",
-        "//tensorflow/core/kernels:linalg",
-        "//tensorflow/core/kernels:list_kernels",
-        "//tensorflow/core/kernels:lookup",
-        "//tensorflow/core/kernels:logging",
-        "//tensorflow/core/kernels:manip",
-        "//tensorflow/core/kernels:math",
-        "//tensorflow/core/kernels:multinomial_op",
-        "//tensorflow/core/kernels:nn",
-        "//tensorflow/core/kernels:parameterized_truncated_normal_op",
-        "//tensorflow/core/kernels:parsing",
-        "//tensorflow/core/kernels:partitioned_function_ops",
-        "//tensorflow/core/kernels:ragged_ops",
-        "//tensorflow/core/kernels:random_ops",
-        "//tensorflow/core/kernels:random_poisson_op",
-        "//tensorflow/core/kernels:remote_fused_graph_ops",
-        "//tensorflow/core/kernels:required",
-        "//tensorflow/core/kernels:resource_variable_ops",
-        "//tensorflow/core/kernels:rpc_op",
-        "//tensorflow/core/kernels:scoped_allocator_ops",
-        "//tensorflow/core/kernels:sdca_ops",
-        "//tensorflow/core/kernels:searchsorted_op",
-        "//tensorflow/core/kernels:set_kernels",
-        "//tensorflow/core/kernels:sparse",
-        "//tensorflow/core/kernels:state",
-        "//tensorflow/core/kernels:stateless_random_ops",
-        "//tensorflow/core/kernels:string",
-        "//tensorflow/core/kernels:summary_kernels",
-        "//tensorflow/core/kernels:training_ops",
-        "//tensorflow/core/kernels:word2vec_kernels",
-    ] + tf_additional_cloud_kernel_deps() + if_not_tx2_llvm_or_windows_cuda([
-        "//tensorflow/core/kernels:nccl_kernels",
-    ]) + if_not_windows([
+               "//tensorflow/core/kernels:array",
```

Comment:
@gunan,

I ran buildifier on the file and it introduced this indentation change. I would have preferred the old way but I didn't revert it back since I thought running buildifier on modified BUILD files is a must. I can revert them back but I expect the buildifier to complain about it. What do you want me to do?

Reply:
Let's revert these lines. @yifeif to check why buildifier modified these lines, but nothing else (in case she knows anything about this)

Reply:
@gunan,
Done. Thanks for the review.

Reply:
Unfortunately, I was wrong and you were right. buildifier checks are now complaining about this.
Sorry for the extra roundtrip caused by this, but lets run buildozer on this again.